errors/messages.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+const (
+	ErrMsgDomain = "dapr.io"
+
+	errStringFormat = "api error: code = %s desc = %s"
+
+	typeGoogleAPI = "type.googleapis.com/"
+
+	// Messages
+	MsgStateGet        = "fail to get %s from state store %s: %s"
```

Comment:
nit: why is this one in the present tense?

Reply:
@elena-kolevska can you confirm? I think it may be due to keeping the legacy messages that currently exist in dapr

Reply:
Yes! This is an existing message we moved from dapr/dapr. I think it shouldn't be a problem if we change an error message, as long as we keep the error codes the same. If you agree, I will update this to match the other error strings.

Reply:
I'll rm these